### ISSUE TITLE
How to set different levels in Log4JS for appenders?

### ISSUE THREAD

**Issue by nonamez** [nomiddlename/log4js-node#479]
Currently my config looks like this:

    log4js.configure({
    	appenders: [
    		{
    			type: 'dateFile',
    			filename: path.join(config.path.root, 'logs/log.log'),
    			pattern: '-yyyy-MM-dd',
    			alwaysIncludePattern: true,
    			category: 'category1'
    		},
    		{
    			type: 'console',
    			category: 'category2'
    		}
    	],
    	levels: {'category1': 'ALL', 'category2': 'DEBUG'}
    })

Basically I want to see everything in file and only info in console. Currently it doesn't show anything. If I remove `category` properties from `appenders` it shows same in both `appenders`.

Version 1.

**Comment by nomiddlename** [comment]
I think you might need the `logLevelFilter`. Something like this:
```javascript
log4js.configure({
  appenders: [
    { type: 'dateFile', .... },
    { type: 'logLevelFilter', level: 'INFO', appender: { type: 'console' } }
]
});
```
That config should get you all logs going to the file, but only INFO and above going to console.

**Comment by nomiddlename** [comment]
Closing this, assuming it fixed the problem for the user.

**Comment by Creative-Difficulty** [comment]
@nomiddlename Is there an updated solution for v6.5.2? I tried converting your solution to v6.5.2 but it throws an error.
Thanks for any advice!

**Comment by lamweili** [comment]
Please refer to the [Log Level Filter](https://log4js-node.github.io/log4js-node/logLevelFilter.html).
